projects/plugins/jetpack/extensions/blocks/subscriptions/menu.js

```diff
@@ -48,18 +51,16 @@ const NewsletterMenu = () => {
 								) }
 							</p>
 						) }
-						<Button
-							onClick={ openModal }
-							style={ {
-								marginRight: '18px',
-								marginTop: '10px',
-							} }
-							variant="secondary"
-							disabled={ isPublished }
-						>
-							{ __( 'Preview email', 'jetpack' ) }
-						</Button>
+						<HStack>
+							<Button onClick={ openModal } variant="secondary" disabled={ isPublished }>
+								{ __( 'Preview email', 'jetpack' ) }
+							</Button>
+							<Button onClick={ openEmailPreview } variant="secondary" disabled={ isPublished }>
```

Comment:
`openEmailPreview` seems to be confusing here. Should er rename this function to be be called `openSendTestEmail` or something like this?

Reply:
good call